<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fitting_tab</class>
 <widget class="QWidget" name="fitting_tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="2">
    <widget class="QLabel" name="filter_by_label">
     <property name="text">
      <string>Filter by:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QComboBox" name="data_type_combo_box">
     <property name="toolTip">
      <string>Select the type of data you want to sequentially fit</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="previous_fit_values_radio">
     <property name="text">
      <string>Parameters from previous fit</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="fit_selected_button">
     <property name="text">
      <string>Fit selected</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="5">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="tableLayout"/>
     </widget>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QCheckBox" name="copy_fit_checkbox">
     <property name="text">
      <string>Copy fit parameters to all</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="seq_fit_button">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Sequentially fit all</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QRadioButton" name="initial_fit_values_radio">
     <property name="text">
      <string>Initial parameters for each fit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
